src/web/app/pages-instructor/instructor-session-individual-extension-page/individual-extension-confirm-modal/individual-extension-confirm-modal.component.html

```diff
@@ -0,0 +1,169 @@
+<fieldset [disabled]="isSubmitting">
```

Comment:
I'm not sure if there's a better way, but I used fieldset so that I could disable all of the buttons at once instead of individually adding the disabled option.

Reply:
I guess it's fine, seems to be allowed by html spec.